db/worker_jobs_migrate/20210809201322_create_good_jobs_separate_database.rb

```diff
@@ -0,0 +1,28 @@
+# same as 20210804180216_create_good_jobs.rb but runs in the worker database
+class CreateGoodJobsSeparateDatabase < ActiveRecord::Migration[5.2]
```

Comment:
I don't know the funcitonal difference, but should this be a `ActiveRecord::Migration[6.1]` instead of `ActiveRecord::Migration[5.2]`?

```suggestion
class CreateGoodJobsSeparateDatabase < ActiveRecord::Migration[6.1]
```

Reply:
This was autogenerated by good_job gem, and their template has 5.2, so seems ok to me to keep?

https://github.com/bensheldon/good_job/blob/main/lib/generators/good_job/templates/install/migrations/create_good_jobs.rb.erb

Reply:
ah ok, 👍🏼